web/src/test/java/com/garret/movies/MovieApplicationTest.java

```diff
@@ -0,0 +1,23 @@
+package com.garret.movies;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.ApplicationContext;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class MovieApplicationTest {
+
+    @Autowired
+    private ApplicationContext context;
+
+    @Test
+    public void initContext() {
+        assertThat(context).isNotNull();
```

Comment:
I don't think you need that, you can remove this assert and autowire of context, just print log that context initialized, it will fail on any error anyway